_tutorial/06-gulp-complete.md

```diff
@@ -3,7 +3,10 @@ layout: default
 title: Complete Gulp Workflow
 step: 6
 ---
-To bring this all together, we need to get Gulp to watch our `index.jsx` file and call our `jsx` task whenever the file is touched. Then we run Gulp once and freely edit the `index.jsx` file and have the changes pick up automatically.
+To bring this all together, we need to get Gulp to watch our `index.jsx` file and call our `jsx` task whenever the file is touched. Then we'll be able to run Gulp once and freely edit the `index.jsx` file and have the changes pick up automatically. First you'll need to install Gulp Watch:
+
+1. 'npm install gulp-watch --save-dev'
```

Comment:
Use a back tick here instead of a single quote - the back tick ( ` ) will format this as code.